packages/flex-plugin-scripts/src/scripts/start.ts

```diff
@@ -34,38 +40,45 @@ const start = async (...args: string[]) => {
   // Future  node version will silently consume unhandled exception
   process.on('unhandledRejection', err => { throw err; });
 
+  const userInputPlugins = _parseUserInputPlugins(...args);
+  const localPlugin = userInputPlugins.find(p => !p.remote);
+  let plugin: FlexConfigurationPlugin | undefined;
+
+  if (localPlugin) {
+    plugin = readPluginsJson().plugins.find((p) => p.name === localPlugin.name);
+  }
+
   let type = WebpackType.Complete;
   if (args[0] === 'flex') {
     type = WebpackType.Static;
 
+    if (!plugin) {
```

Comment:
Based on your change to `index.ts` shouldn't we always have a plugin now? i..e this should always throw an exception, not just for `WebpackType.Static`

Reply:
yes agreed will move it out